test/load.test.ts

```diff
@@ -591,6 +610,10 @@ describe('#mergeDependency()', () => {
             date: '20200413230227'
           }
         };
+      } else if (uri === 'https://packages.fhir.org/hl7.fhir.r4.core') {
+        return { data: TERM_PKG_RESPONSE };
+      } else if (uri === 'https://packages2.fhir.org/hl7.fhir.r4.core') {
+        return { data: EXT_PKG_RESPONSE };
```

Comment:
Similar to above, I'd prefer that the response match the query -- so either use uri `https://packages2.fhir.org/packages/hl7.fhir.uv.extensions` or introduce a matching response for R4 Core.